### PR TITLE
tak

### PR DESCRIPTION
**Pull request by ArsBeneMoriendi**

### CODE REVIEW

File: i18n/en-US.json [hunk 1]
```diff
@@ -10,22 +10,22 @@
...
     "BAN_USER": "Banned",
     "PUNISHMENT_ID": "Punishment ID",
     "BAN_REASON": "Banned by {{- user}} for {{- reason}}",
     "REASON": "Reason",
     "NONE": "None",
     "REASON_LONGER_THAN_1024": "Reason can't be longer than 1024 characters",
-    "HYPIXEL_INVALID_PLAYER": "You provided an invalid user!",
-    "HYPIXEL_NO_ARGS": "You need to provide a player! Correct use: {{- command}} <player>",
+    "HYPIXEL_INVALID_PLAYER": "You've provided an invalid user!",
+    "HYPIXEL_NO_ARGS": "You need to provide a player! Correct usege: {{- command}} <player>",
```
Comment (hkubus) [inline i18n/en-US.json]:
```suggestion
    "HYPIXEL_NO_ARGS": "You need to provide a player! Correct usage: {{- command}} <player>",
```
epic fail

**Review by hkubus — CHANGES_REQUESTED** [review]

**Review by hkubus — APPROVED** [review]

**Review by SuperchupuDev — APPROVED** [review]

File: i18n/en-US.json [hunk 2]
```diff
@@ -10,22 +10,22 @@
...
-    "BAN_BOT_PERMISSIONS_MISSING": "I can't ban this member, because I don't have the permissions needed to ban!",
+    "BAN_PERMISSIONS_MISSING": "You don't have permissions needed to ban a member!",
+    "BAN_BOT_PERMISSIONS_MISSING": "I can't ban this member, because I don't have permissions needed to ban!",
     "BAN_ARGUMENTS_MISSING": "You haven't provided a user to ban!",
-    "BAN_MEMBER_MISSING": "The member that you want to ban doesn't exist!",
+    "BAN_MEMBER_MISSING": "The member you want to ban doesn't exist!",
     "BAN_ERROR_YOURSELF": "You can't ban yourself!",
     "BAN_SUCCESS_TITLE": "Banned!",
-    "BAN_SUCCESS_DESCRIPTION": "Successfully banned a member!",
+    "BAN_SUCCESS_DESCRIPTION": "Successfully banned the member!",
```
Comment (Nyameliaaaa) [inline i18n/en-US.json:19]:
```suggestion
    "BAN_SUCCESS_DESCRIPTION": "Successfully banned that member!",
```

File: i18n/en-US.json [hunk 3]
```diff
@@ -54,27 +54,27 @@
...
-    "KICK_BOT_PERMISSIONS_MISSING": "I can't kick this member, because I don't have the permissions needed to kick!",
+    "KICK_PERMISSIONS_MISSING": "You don't have permissions needed to kick a member!",
+    "KICK_BOT_PERMISSIONS_MISSING": "I can't kick this member, because I don't have permissions needed to kick!",
     "KICK_ARGUMENTS_MISSING": "You haven't provided a user to kick!",
-    "KICK_MEMBER_MISSING": "The member that you want to kick doesn't exist!",
+    "KICK_MEMBER_MISSING": "The member you want to kick doesn't exist!",
     "KICK_ERROR_YOURSELF": "You can't kick yourself!",
     "KICK_SUCCESS_TITLE": "Kicked!",
-    "KICK_SUCCESS_DESCRIPTION": "Successfully kicked a member!",
+    "KICK_SUCCESS_DESCRIPTION": "Successfully kicked the member!",
```
Comment (Nyameliaaaa) [inline i18n/en-US.json:63]:
```suggestion
    "KICK_SUCCESS_DESCRIPTION": "Successfully kicked that member!",
```

File: i18n/en-US.json [hunk 4]
```diff
@@ -54,27 +54,27 @@
...
     "LISTCASE_NEXT": "Next",
-    "MUTE_BOT_PERMISSIONS_MISSING": "I can't mute this member, because I don't have the permissions needed to mute!",
-    "MUTE_PERMISSIONS_MISSING": "You don't have the permissions needed to mute a member!",
-    "MUTE_MEMBER_MISSING": "The member that you want to mute doesn't exist!",
+    "MUTE_BOT_PERMISSIONS_MISSING": "I can't mute this member, because I don't have permissions needed to mute!",
+    "MUTE_PERMISSIONS_MISSING": "You don't have permissions needed to mute a member!",
+    "MUTE_MEMBER_MISSING": "The member you want to mute doesn't exist!",
     "MUTE_SUCCESS_TITLE": "Muted!",
-    "MUTE_SUCCESS_DESCRIPTION": "Succesfully muted a member!",
+    "MUTE_SUCCESS_DESCRIPTION": "Succesfully muted the member!",
```
Comment (Nyameliaaaa) [inline i18n/en-US.json:77]:
```suggestion
    "MUTE_SUCCESS_DESCRIPTION": "Succesfully muted that member!",
```

File: i18n/en-US.json [hunk 5]
```diff
@@ -95,20 +95,20 @@
...
     "STATS_UPTIME": "Uptime",
-    "UNMUTE_BOT_PERMISSIONS_MISSING": "I can't unmute this member, because I don't have the permissions needed to unmute!",
-    "UNMUTE_PERMISSIONS_MISSING": "You don't have the permissions needed to unmute a member!",
-    "UNMUTE_MEMBER_MISSING": "The member that you want to unmute doesn't exist!",
+    "UNMUTE_BOT_PERMISSIONS_MISSING": "I can't unmute this member, because I don't have permissions needed to unmute!",
+    "UNMUTE_PERMISSIONS_MISSING": "You don't have permissions needed to unmute a member!",
+    "UNMUTE_MEMBER_MISSING": "The member you want to unmute doesn't exist!",
     "UNMUTE_SUCCESS_TITLE": "Unmuted!",
-    "UNMUTE_SUCCESS_DESCRIPTION": "Succesfully unmuted a member!",
+    "UNMUTE_SUCCESS_DESCRIPTION": "Succesfully unmuted the member!",
```
Comment (Nyameliaaaa) [inline i18n/en-US.json:102]:
```suggestion
    "UNMUTE_SUCCESS_DESCRIPTION": "Succesfully unmuted that member!",
```

File: i18n/en-US.json [hunk 6]
```diff
@@ -95,20 +95,20 @@
...
     "UNMUTE_REASON": "Unmuted by {{- user}} for {{- reason}}",
-    "WARN_PERMISSIONS_MISSING": "You don't have the permissions needed to warn a member!",
+    "WARN_PERMISSIONS_MISSING": "You don't have permissions needed to warn a member!",
     "WARN_ARGUMENTS_MISSING": "You haven't provided a member to warn!",
-    "WARN_MEMBER_MISSING": "The member that you want to warn doesn't exist!",
+    "WARN_MEMBER_MISSING": "The member you want to warn doesn't exist!",
     "WARN_ERROR_YOURSELF": "You can't warn yourself!",
     "WARN_SUCCESS_TITLE": "Warned!",
-    "WARN_SUCCESS_DESCRIPTION": "Successfully warned a member!",
+    "WARN_SUCCESS_DESCRIPTION": "Successfully warned the member!",
```
Comment (Nyameliaaaa) [inline i18n/en-US.json:111]:
```suggestion
    "WARN_SUCCESS_DESCRIPTION": "Successfully warned that member!",
```

File: i18n/en-US.json [hunk 7]
```diff
@@ -121,17 +121,17 @@
...
     "HOWGAY": "🏳️‍🌈 {{- thing}} is {{- percent}}% gay",
     "HOWGAY_ARGS": "You haven't provided a thing to check!",
     "AI_TITLE": "Generated!",
     "AI_DESCRIPTION": "Your text: {{- text}}",
     "AI_ERROR": "Our AI is currently down, try again in a few minutes!",
-    "AI_LIMIT": "You can only have one AI request in the queue at a time!",
+    "AI_LIMIT": "You can have only one AI request in the queue at the time!",
     "AI_ARGS": "You haven't provided a text to generate!",
-    "AI_WAIT": "Added to queue! You're number {{- queue}} in queue",
+    "AI_WAIT": "Added to queue! You're a number {{- queue}} in the queue",
```
Comment (Nyameliaaaa) [inline i18n/en-US.json:134]:
```suggestion
    "AI_WAIT": "Added to queue! You're number {{- queue}} in the queue",
```

**Review by Nyameliaaaa — CHANGES_REQUESTED** [review]
idk about the polish but yeah

**Comment by Nyameliaaaa** [comment]
nvm im an idiot it got merged

**Review by Nyameliaaaa — COMMENTED** [review]
sex

**Comment by luczoxz** [comment]
penis

**Pull request merged**